mixin/signals/overview.libsonnet

```diff
@@ -300,5 +300,33 @@ function(this) {
         },
       },
     },
+
+    computeCreditUsagePercentage: {
+      name: 'Compute credit usage percentage',
+      nameShort: 'Compute % of limit',
+      description: 'Percentage of compute credit limit being used.',
+      type: 'raw',
+      unit: 'percent',
+      sources: {
+        prometheus: {
+          expr: 'sum by (' + aggregationLabels + ') (last_over_time(snowflake_used_compute_credits{%(queriesSelector)s}[24h]) / ' + this.alertsComputeCreditUsageLimit + ')',
```

Comment:
This has been changed since my last review, same as my other comment on the alerts.

Looks like there's some inconsistency in the use of the used credit metrics, with both [1h] and [24h] windows used. 
Don't think we need to be worried about missing data in a short window, as that in itself should be alerted on. I'd prefer if we align on 1hr intervals for this (even if the underlying datasource only updates every ~6h, that's transient to us)

Reply:
Good grief I've already forgotten about that PR :upside_down_face: 

Thank you, yes that's right. Just consistency wise could you add a comment above these why it's different in dashboard signals versus alerts?